reana/cli.py

```diff
@@ -195,6 +196,18 @@ def select_components(components):
     return list(output)
 
 
+def is_component_dockerisable(component):
```

Comment:
Would be `is_component_dockerized` more clear? I would expect this function to tell me whether I can make the component Docker compatible or not.